system/security/LdapSecurity/aci.cpp

```diff
@@ -972,7 +976,7 @@ void AciProcessor::cacheSid(const char* name, int len, const void* sidbuf)
 void AciProcessor::lookupSid(const char* act_name, MemoryBuffer& act_sid, ACT_TYPE acttype)
```

Comment:
It's part of the  IPermissionProcessor pure virtual base class, which is needed in derived class PermissionProcessor but not here